Bel 4204 More Changes to Move Canvas to Common Infrastructure

JoshuaGleason opened the pull request:
[Link to Jira ticket](https://strongmind.atlassian.net/browse/BEL-4204)

## Purpose 
<!-- what/why -->
More changes to move canvas to common infrastructure
## Approach 
<!-- how -->
- Build s3 policy despite storage=false
- Namespace the certificate
- Some ignore changes for cluster and clusterinstance
## Testing
<!-- what did you do to confirm this works/what would a QA engineer do to confirm - Think: setup process, steps, expected outcomes -->
All changes have been used to deploy new-id-sandbox and cdcr locally

markng commented:
Got any tests for these? (And I'm guessing we had no coverage beforehand, or this would have caused some failures?)

JoshuaGleason commented:
I added a test for the certificate namespace.
I remove the conditional on the test for the s3 policy as it apparently isn't working as intended.
The reason why we removed the storage=True conditional on the policy is b/c storage=True tries to create a bucket and we weren't able to get the bucket to import at the right layer, but we still need the policy for the container.
